evasquez26/angular4-startup
# S1-ONE-107 others profile

## Story / Task

see [ONE-107](https://bixlabs.atlassian.net/browse/ONE-107)


## What this PR does?

Adds functionality to see other user's profile

### Before 

No user info for other users

### After

You can see another user's profile info

### How to test / reproduce
- Based on #11 data you should be able to do the following:
- Click "Followers"
- See some user list
- Click in a follower
- See it loads user data

### TODOs
- [x] Rebase after merginf #11 
- [ ] Use real API

### Related PRs
N/A

## Migrations

NO


Thank you, @evasquez26
